examples/protocols/multicolor-particle-calibration/artifacts/multicolor-particle-calibration.md

```diff
@@ -24,47 +24,47 @@ Adapted from [https://dx.doi.org/10.17504/protocols.io.bht7j6rn](https://dx.doi.
-27. Perform a series of 10 2-fold dilutions on `D1` using `Phosphate Buffered Saline` as diluent to a final volume of 200.0 microliter in  wells ['D1', 'D2', 'D3', 'D4', 'D5', 'D6', 'D7', 'D8', 'D9', 'D10', 'D11'] of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-28. Perform a series of 10 2-fold dilutions on `E1` using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in  wells ['E1', 'E2', 'E3', 'E4', 'E5', 'E6', 'E7', 'E8', 'E9', 'E10', 'E11'] of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-29. Perform a series of 10 2-fold dilutions on `F1` using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in  wells ['F1', 'F2', 'F3', 'F4', 'F5', 'F6', 'F7', 'F8', 'F9', 'F10', 'F11'] of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-30. Perform a series of 10 2-fold dilutions on `G1` using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in  wells ['G1', 'G2', 'G3', 'G4', 'G5', 'G6', 'G7', 'G8', 'G9', 'G10', 'G11'] of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-31. Perform a series of 10 2-fold dilutions on `H1` using `Water, sterile-filtered, BioReagent, suitable for cell culture` as diluent to a final volume of 200.0 microliter in  wells ['H1', 'H2', 'H3', 'H4', 'H5', 'H6', 'H7', 'H8', 'H9', 'H10', 'H11'] of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
-32. Discard 100.0 microliter from wells ['A11', 'B11', 'C11', 'D11', 'E11', 'F11', 'G11', 'H11'] of 96 well plate `calibration plate`.  This step ensures that all wells contain an equivalent volume. Be sure to change pipette tips for every well to avoid cross-contamination
-33. Transfer 100.0 microliter of `Phosphate Buffered Saline` sample to wells A1:H12 of  96 well plate `calibration plate`.  This will bring all wells to volume 200 microliter.
-34. Transfer 100.0 microliter of `Water, sterile-filtered, BioReagent, suitable for cell culture` sample to wells A1:H12 of  96 well plate `calibration plate`.  This will bring all wells to volume 200 microliter.
+25. Perform a series of 10 2-fold dilutions on `B` using `Phosphate Buffered Saline` as diluent to a final volume of 200.0 microliter in  wells B1:B11 of 96 well plate `calibration plate`.  For each transfer, pipette up and down 3X to ensure the dilution is mixed homogeneously.
```

Comment:
Could say "row B"?  Also the well coordinate is displayed as inline code `B` instead of just regular text -- I think this is the only place where a well coordinate is displayed as such.  I don't have a strong preference either way, just so long as we are consistent.